pkgs/by-name/te/television/package.nix

```diff
@@ -1,24 +1,36 @@
 {
   lib,
+  stdenv,
   rustPlatform,
   fetchFromGitHub,
+  versionCheckHook,
+
+  # passthru
   testers,
   television,
   nix-update-script,
 }:
 rustPlatform.buildRustPackage rec {
   pname = "television";
-  version = "0.10.4";
+  version = "0.10.6";
 
   src = fetchFromGitHub {
     owner = "alexpasmantier";
     repo = "television";
     tag = version;
-    hash = "sha256-ja3Xqp8nRTQnf0K1okHSBPcqQe/m8DqH7UWbdohxlvM=";
+    hash = "sha256-O5BV3+zOUnDdviA8tCGxYmPTTBfge5+BKolIgJSqgOI=";
   };
 
   useFetchCargoVendor = true;
-  cargoHash = "sha256-eplkAaNgoAxMLK3BG0EvNLYT1T3vJpHpbuGvwooegeI=";
+  cargoHash = "sha256-4ApdtAuVQNeBBgBhXpp/N3CMLTZKPJnI3vUiAQUddyg=";
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${placeholder "out"}/bin/tv";
+  versionCheckProgramArg = [ "--version" ];
+  # Fails on darwin with 'Error: Operation not permitted (os error 1)'
+  doInstallCheck = !stdenv.hostPlatform.isDarwin;
```

Comment:
```suggestion
```
This is completely unnecessary. We already use `testers.testVersion`, and this doesn't even support Darwin